### DISCUSSION TITLE
Video Style Transfer

### DISCUSSION THREAD

**Discussion by laetokang** [huggingface/diffusers#8657] [Q&A]
Hello!

I am looking to create videos in different styles (for example, transforming them into a cartoon style) using video footage as input. Could you please recommend any models or repositories that would be helpful for this task?

Thank you!

**Comment by tolgacangoz** [comment]
Hi @laetokang,
There is [`AnimateDiffVideoToVideoPipeline`](https://huggingface.co/docs/diffusers/main/en/api/pipelines/animatediff#animatediffvideotovideopipeline) implemented by @a-r-r-o-w.
Also [FRESCO](https://github.com/huggingface/diffusers/issues/7408). It was implemented by its author as [a community pipeline](https://github.com/huggingface/diffusers/tree/main/examples/community#fresco).
